bin/bunyan

```diff
@@ -11,7 +11,7 @@
  * vim: expandtab:ts=4:sw=4
  */
 
-var VERSION = '1.8.1';
+var VERSION = '1.8.0';
```

Comment:
Thank you. It was my mistake. I've updated it and removed extra indenting as you suggested.